blog-single-36.html

```diff
@@ -0,0 +1,247 @@
+		<div class="overlay-control background-90-f">
+			<a class="cross" href="#"></a>
+		</div>
+	</div>
+	<div class="overlay-window credits-overlay colors-g background-95-g">
+		<div class="overlay-control background-85-d">
+			<a class="cross" href="#"></a>
+		</div>
+	</div>
+	<!-- <div class="gate colors-e background-solid">
+			<div class="gate-bar background-heading-a"></div>
+			<ul class="loader">
+				<li class="background-100-g"></li>
+				<li class="background-100-b"></li>
+				<li class="background-100-f"></li>
+			</ul>
+		</div> -->
+	<!-- BEGIN: Page content -->
+	<!-- <img alt class="bg" src="images/Background.png" /><img alt class="bg" src="images/Background.png" /> -->
+	<section>
+		<div class="view">
+			<div class="content  colors-e background-90-e">
+				<div class="container">
+					<div class="row" style="margin-top: 5%">
+						<div class="col-md-9">
+							<div class="row post">
+								<div class="col-md-3 post-meta">
+									<div class="post-day">21</div>
+									<div class="post-month">July 2020</div>
+								</div>
+								<div class="col-md-9">
+									<div class="post-body background-75-e">
+                                        <h3><span class="highlight">GG Web Portals</span></h3><br>
+                                        <p>
+                                            Geo Gecko has developed numerous web portals with mapping and analytical functionality. They're built in open source code, are typically easy-to/automatically updated and don't require technical expertise to operate.<br>
+                                            Click the below images to interact more.
+                                        </p>
+                                        <h5>Pan African COVID 19 Analysis</h5>
+										<div class="post-image">
+                                            <a target="_blank" href="https://www.africacorona.com/"><img class="fluid-width" src="images/AfricaCorona.png"
+											alt="Post with Featured Image"></a>
+										</div>
+										<div class="image-caption">
+                                        <p>Pan African COVID 19 Analysis</p>
+										</div><br>
+
+                                        <h5>GVTC Geonode</h5>
+										<div class="post-image">
+                                            <a target="_blank" href="http://www.gvtcgeonode.com/"><img class="fluid-width" src="images/GeoNode.png"
+											alt="Post with Featured Image"></a>
+										</div>
+										<div class="image-caption">
+                                        <p>An open source platform for sharing geospatial data and maps</p>
+										</div><br>
+
+                                        <h5>GVTC Webmap</h5>
+										<div class="post-image">
+                                            <a target="_blank" href="https://geo-gecko.github.io/GVTC_Webmaps/orientationpage.html/"><img class="fluid-width" src="images/GVTC.png"
```

Comment:
Also, please remove the last slash in the URL here because it leads to a 404.

Reply:
![Screenshot (97)](https://user-images.githubusercontent.com/25029435/88411090-0183ba80-cde0-11ea-9629-725264a3a1ae.png)
moved recent posts to top right